js/numbers.js

```diff
@@ -0,0 +1,46 @@
+function check1()
```

Comment:
Most of the comments added here `https://github.com/g-strzelecki/hangman-the-game/pull/1` is also valid for this PR.